Multisig/Logic/Models/Safe.swift

```diff
@@ -15,6 +15,31 @@ extension Safe: Identifiable {
         self.createdAt = Date()
     }
 
+    static func download(at address: String) throws {
+        _ = try App.shared.safeRelayService.safeInfo(at: address)
```

Comment:
So we don't save Safe meta like master copy and etc. I didn't see it in this PR, or I missed it.

Reply:
In this PR I don't do this. I decided to just check the name without pulling the data into a database, so that we just save when the whole form completes.

We will pull this information again on the safe saving.

Reply:
This is because doing like this seems to be simpler (no need to handle transitive state of the database context)

Reply:
So you mean we will implement meta info logic in the next sprint, right?

Reply:
yes